js/dactylo.js

```diff
@@ -283,9 +276,24 @@ window.addEventListener('DOMContentLoaded', () => {
   gQuacks.addEventListener('dblclick', moreQuacks); // cheat code!
 
   // startup
-  Promise.all([fetchNgrams(), fetchWords(), fetchLayout()])
-    .then(setLessonLevel);
+  const loadLayout = () => {
+    gLayout.value = window.location.hash.split('/')[1];
```

Comment:
Wait ,what ? Why do you split on a slash ?

Reply:
Oh ’cause I thought we would have the leading slash in the url hash, like on the stats page

Reply:
TBH I’m surprised this could even work. :-)

I think the `.slice(1)` approach you used before is saner as I expect the hash to **always** start with a `#` sign and never contain any slash, but as long as it works…